gamestonk_terminal/cryptocurrency/discovery/coinmarketcap_model.py

```diff
@@ -0,0 +1,40 @@
+"""CoinMarketCap model"""
+__docformat__ = "numpy"
+
+import pandas as pd
+from coinmarketcapapi import CoinMarketCapAPI
+import gamestonk_terminal.config_terminal as cfg
+
+
+def get_cmc_top_n() -> pd.DataFrame:
+    """Shows top n coins. [Source: CoinMarketCap]
```

Comment:
@DidierRLopes do we need the [Source: {}] if it is already in that /source/ menu?

Reply:
That's a good question. But I think is good practice because if one day we want to change the name of the files, we still know the command uses data from x